From 1b3902f30acbe80bce751eb9d4ff3a793f57833c Mon Sep 17 00:00:00 2001
From: Thomas Duckworth <tduck973564@gmail.com>
Date: Sun, 16 Jun 2024 19:34:46 +1000
Subject: [PATCH] Change icon

---
 dnfdragora/ui.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dnfdragora/ui.py b/dnfdragora/ui.py
index 63c8ede..2a66a5b 100644
--- a/dnfdragora/ui.py
+++ b/dnfdragora/ui.py
@@ -386,8 +386,8 @@ class mainGui(dnfdragora.basedragora.BaseDragora):
         '''
         yui.YUI.app().setApplicationTitle(_("Software Management - dnfdragora"))
 
-        self.icon = self.images_path + "dnfdragora.png"
-        self.logo = self.images_path + "dnfdragora-logo.png"
+        self.icon = "muon"
+        self.logo = "muon"
         yui.YUI.app().setApplicationIcon(self.icon)
 
         MGAPlugin = "mga"
-- 
2.45.2

